smrf/distribute/soil_temp.py

```diff
@@ -37,6 +39,8 @@ class ts(image_data.image_data):
     # be written during main distribute loop
     post_process_variables = {}
 
+    BASE_THREAD_VARIABLES = []
```

Comment:
No need to declare the empty variable again, since it is declared in the parent class `image_data`.
There are more cases below.

Reply:
Good catch. Soil temp isn't added to the queue like others since it's constant.

Reply:
Actually this is needed as `_initialize` isn't called for soil temp.

Reply:
Any thoughts on this? I forgot to hit the final 'comment' button. Hopefully you received a notification by now.

Reply:
These are tough because while they are constant, some variables need to be added to the queue dependent on configuration. So the problem I was running into in the tests if you add a variable to the base variable list, it will modify the class and not the instance. That's why I call `_intialize` to create another property that can be modified on the instance. Open to other ideas though as it still seems to be slightly problematic.

Reply:
Can we merge this onto a branch instead, so I can do a quick sample implementation? Otherwise I would have to fork your fork to do this.

Reply:
It's up on the main repo.